src/Templates/TodoTemplate/Bit.TodoTemplate/Web/Shared/TodoTemplateErrorBoundary.razor.cs

```diff
@@ -5,9 +5,9 @@ public partial class TodoTemplateErrorBoundary
 {
     private bool ShowException { get; set; }
 
-    [AutoInject] private IExceptionHandler ExceptionHandler { get; set; } = default!;
+    [AutoInject] private IExceptionHandler ExceptionHandler = default!;
```

Comment:
Use field name convention for all fields @sajadkardel

Reply:
`customerService` or `_customerService` ?

Reply:
I'd prefer `customerService`